context/app/api/vitessce_confs/base_confs.py

```diff
@@ -177,39 +177,33 @@ def _setup_scatterplot_view_config(self, vc, dataset):
 
 class SPRMViewConfBuilder(ImagePyramidViewConfBuilder):
 
-    def _get_sprm_image(self):
-        image_file_regex = f"{self._imaging_path}/{self._base_name}.ome.tiff?"
+    def _make_full_image_path(self):
```

Comment:
Could this just be `_get`? One challenge for me as I get familiar with the code is understanding what the methods do, and whether they have side effects.